mobileapp/tutorials/localhost_with_docker.mdx

```diff
@@ -0,0 +1,249 @@
+---
+title: Getting up & running. The fast way.
+sidebar_label: The fast track
+---
+
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+
+:::caution Prerequisites
+In this tutorial we assume that you are comfortable working with [Docker](https://www.docker.com/), [Git](https://git-scm.com/)
+and the Terminal. In addition to this you would need a [Firebase](https://firebase.google.com/) account and your dev. machine should be configured to work
+with [React-Native](https://reactnative.dev) projects.
+:::
+
+:::warning
+The tutorial was tested on a Mac. You should be able to run the Android version of the app on Linux or Windows, but some
+things might be different.
+:::
+
+The Trustline app relies on a [Relay](/relay/introduction) server for its communication with the Trustlines blockchain (TLBC).
+Especially at the beginning, when you just want to get started, it is a daunting task to set up everything.
+In this tutorial we'll take a shortcut. Instead of setting up a Relay server that will communicate with the TLBC chain or with
+the Laika test chain, we'll use a utility that the maintainers of the [Trustlines Protocol use for e2e tests](https://github.com/trustlines-protocol/end2end).
+
+The tool in the end2end repository relies on Docker and sets the followings:
+
+- A development OpenEthereum node that automatically mines a block when it receives a transaction
+- A relay server
+- A py-eth-index instance to index events for the relay
+- A contracts docker image that will deploy test currency networks, exchanges, and identity contracts
+
+:::info
+By using the end2end utility you won't be sending transactions to the TLBC or Laika chains, but instead to a local
+OpenEthereum node. This way you don't have to pay for transactions when you get started. At a later stage when you
```

Comment:
"but instead to a local dev chain." would be more correct.

Because you could send your tx to a local OE node that is connected to tlbc/laika and the node forwards the tx.